blocks/shared-styles/_children/byline/index.scss

```diff
@@ -0,0 +1,31 @@
+.byline {
+  $self: &;
+  color: #191919;
+  font-size: 1.25rem;
+  line-height: 2.5rem;
+
+  &--list {
+    display: inline;
+    font-size: 0.875rem;
+    line-height: 1rem;
+
+    #{$self}__by {
+      color: #3b3b3b;
+      margin-right: 0;
+    }
+
+    #{$self}__names {
```

Comment:
nice, this $self new?

Reply:
No, it's a Sass way to do BEM naming with nesting